lib/stdlib/test/stdlib_bench_SUITE.erl

```diff
@@ -157,41 +162,59 @@ norm_data(Config) ->
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+match_single_pattern_no_match() ->
```

Comment:
This should be an arity 1 function.

Reply:
Ah, that was it! Thank you. Fixed, squashed and pushed.